src/traversal/writeRelayUpdatePayload.js

```diff
@@ -76,6 +76,15 @@ function writeRelayUpdatePayload(
   payload: {[key: string]: mixed},
   {configs, isOptimisticUpdate}: UpdateOptions
 ): void {
+  invariant(
+    typeof payload[CLIENT_MUTATION_ID] === 'string',
+    'writeRelayUpdatePayload: expected clientMutationID on the payload for mutation `%s` to be string' +
+    ', instead received the `%s` `%s`.',
+    operation.getCall().name,
```

Comment:
I tried that and was getting the route name I believe, I'll have to double check